### PR TITLE
[Ready for Review] French - Design and Develop Overview

### PR DESCRIPTION
**Pull request by Soahmed2031**

### CODE REVIEW

File: index.fr.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+* **[Tutoriel : menus](https://www.w3.org/WAI/tutorials/menus/)**
+* **[Tutoriel : images](https://www.w3.org/WAI/tutorials/images/)**
+* **[Tutoriel : tableaux](https://www.w3.org/WAI/tutorials/tables/)**
+* **[Tutoriel : formes](https://www.w3.org/WAI/tutorials/forms/)**
+* **[Tutoriel : carrousels](https://www.w3.org/WAI/tutorials/carousels/)**
+
+## Autres ressources pour la conception et le développement
+
+[Comment répondre aux exigences des WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)
+: Une référence personnalisable aux exigences (“critères de réussite”) et aux techniques des Règles de l’accessibilité des contenus Web (WCAG).
```
Comment (remibetin) [inline index.fr.md:76]: The meaning is not very easy to grasp. "Référence rapide" seems to be used in Canada and Switzerland, but is really used in France.  Suggestion "Un aide-mémoire personnalisable des exigences ("critères de réussite") et des techniques [...]"

File: index.fr.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* **[Tutoriel : structure d’une page](https://www.w3.org/WAI/tutorials/page-structure/)**(en anglais)
+* **[Tutoriel : menus](https://www.w3.org/WAI/tutorials/menus/)**
+* **[Tutoriel : images](https://www.w3.org/WAI/tutorials/images/)**
+* **[Tutoriel : tableaux](https://www.w3.org/WAI/tutorials/tables/)**
+* **[Tutoriel : formes](https://www.w3.org/WAI/tutorials/forms/)**
+* **[Tutoriel : carrousels](https://www.w3.org/WAI/tutorials/carousels/)**
+
+## Autres ressources pour la conception et le développement
+
+[Comment répondre aux exigences des WCAG (référence rapide)](https://www.w3.org/WAI/WCAG21/quickref/)
```
Comment (remibetin) [inline index.fr.md:75]: What do you think of "aide-mémoire" rather than "référence rapide" ? (see comment below)

File: index.fr.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+: Une référence personnalisable aux exigences (“critères de réussite”) et aux techniques des Règles de l’accessibilité des contenus Web (WCAG).
+
+[Pratiques de création WAI-ARIA](https://www.w3.org/TR/wai-aria-practices/)
```
Comment (remibetin) [inline index.fr.md:78]: "Pratiques de création **de** WAI-ARIA" ? (sous-entendu création d'applications accessibles)

File: index.fr.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+: Fournit des approches, des conseils, et des exemples pour aider les développeurs d’applications Web à créer des widgets, une navigation, et des comportements accessibles en utilisant les rôles, les états, et les propriétés WAI-ARIA (Accessible Rich Internet Applications).
```
Comment (remibetin) [inline index.fr.md:79]: Maybe we should use `<i>` around _Accessible Rich Internet Applications_ considering it is in a foreign langage.

**Review by remibetin — CHANGES_REQUESTED** [review]
Thank you @Soahmed2031 for your work on this french translation. Here is my review with suggestions and thoughts.

I also noticed in the Netlify preview that some links to non-translated resources are not displayed with "(en anglais)" at the end, but I don't know why it is sometimes automatically added and sometimes not.

Let me know what you think of my input.

**Comment by slhenry** [comment]
integrated into https://github.com/w3c/wai-design-develop-overview/pull/11

**Pull request closed without merging**